packages/ai-core/src/components/AnalysisResultsContainer.tsx

```diff
@@ -1,25 +1,37 @@
 import {cn, ScrollArea, ScrollBar} from '@sqlrooms/ui';
 import {ChevronDown} from 'lucide-react';
 import React, {useRef} from 'react';
+import {Components} from 'react-markdown';
 import {useStoreWithAi} from '../AiSlice';
 import {useScrollToBottom} from '../hooks/useScrollToBottom';
 import {AnalysisResult} from './AnalysisResult';
 import {AiThinkingDots} from './AiThinkingDots';
 
 export const AnalysisResultsContainer: React.FC<{
   className?: string;
-}> = ({className}) => {
+  enableReasoningBox?: boolean;
+  customMarkdownComponents?: Partial<Components>;
+  userTools?: string[];
+}> = ({
+  className,
+  enableReasoningBox = false,
+  customMarkdownComponents,
+  userTools,
+}) => {
   const isRunningAnalysis = useStoreWithAi((s) => s.ai.isRunningAnalysis);
   const currentAnalysisResults = useStoreWithAi((s) =>
     s.ai.getAnalysisResults(),
   );
+  const uiMessages = useStoreWithAi(
+    (s) => s.ai.getCurrentSession()?.uiMessages || [],
+  );
 
   const containerRef = useRef<HTMLDivElement>(null);
   const endRef = useRef<HTMLDivElement>(null);
   const {showScrollButton, scrollToBottom} = useScrollToBottom({
     containerRef,
     endRef,
-    dataToObserve: currentAnalysisResults,
+    dataToObserve: uiMessages,
```

Comment:
@ilyabo I can't figure out how to show the "Blue scroll down icon" in sqlrooms, can you help to fix it? Thanks!

<img width="505" height="236" alt="Image" src="https://github.com/user-attachments/assets/2a5dca45-b404-424f-b637-3244dd41ae06" />